spec/components/schemas/Instruments/InstrumentResponse.yaml

```
@@ -75,3 +79,20 @@ properties:
         type: string
         description: The issuer/card scheme product type
         example: CLASSIC
+    customer: 
+        type: object
+        description: The customer details
+        properties:
+            id:
+                type: string
+                description: The customer's unique identifier
+                example: "cus_y3oqhf46pyzuxjbcn2giaqnb44"
+            email: 
+                type: string
+                format: email
+                description: The customer's email address
+                example: JohnTest@test.com
+            name:
+                type: string
+                description: The customer's name
+                example: Jack Napier
```

```suggestion
                example: John Test
```
So it matches the email?